datacube/utils/aws/__init__.py

```diff
@@ -479,6 +480,7 @@ def configure_s3_access(
     requester_pays: bool = False,
     cloud_defaults: bool = True,
     client=None,
+    driver: str | None = None,
```

Comment:
Not having a doc string for a single parameter triggers a warning in my PyCharm and this function doesn't look deprecated, can the parameter get a doc string?